src/App.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { describe, it, expect, test, vi } from 'vitest'; // vi have a mock feature
+import { render } from '@testing-library/react';
+import App from './App';
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+
+test('demo', () => {
+  expect(true).toBe(true);
+});
+
+describe('render', () => {
+  
+  it('renders the main page', () => {
+    //const component = ReactTestUtils.renderIntoDocument()
+    const component = render(
+    <MemoryRouter>
+      <App />
+    </MemoryRouter>
+    );
+    expect(true).toBeTruthy();
+    expect(component).toBe(component)
```

Comment:
ten test niczego nie testuje za bardzo. Ale zostaw go, możemy na następnej sesji zrobić go przydatnym. A jeśli chcesz sam się pobawić to należałoby sprawdzić czy jakiś wybrany tekst na stronie głównej wyświetla się po podpięciu komponentu App. Zasada jest taka, że testuje się z punktu widzenia użytkownika i tego jak on chodzi po stronie, sprawdzając to, co na tej stronie widzi i jak strona ma z nim interreagować. Czyli nie testujemy technicznie, a biznesowo. Przykładowo: użytkownika nie interesuje czy komponent się wyrenderował, bo ten komponent mógłby być pusty i wtedy czy się renderuje czy nie to dla użytkownika nie ma znaczenia. Za to użytkownika na pewno interesuje nagłówek na stronie - i to bym tutaj w tym teście sprawdziła.

Reply:
Na ten moment to test testu. Jesteśmy przygotowani na nasze testy, chodziło o ustawienie. This test is only our first settings and check that loading and check cooperation with circlecl. I know ;) expect component to be component.